x-pack/plugins/apm/public/components/app/transaction_details/distribution/index.tsx

```diff
@@ -132,7 +132,7 @@ export function TransactionDistribution({
 
   return (
     <div data-test-subj="apmTransactionDistributionTabContent">
-      <EuiFlexGroup>
+      <EuiFlexGroup style={{ minHeight: 56 }}>
```

Comment:
Can you explain why this is necessary? Could we use an EUI px var instead?

Reply:
Added a comment for it [here](https://github.com/elastic/kibana/pull/109187/commits/be2657501e18cd41c101492c3af014f0c56c4ee2). The small size selection text is causing the width of that title box to be smaller than the other two (which have the help button), so when you navigate between the tabs there's a bit of a "flickering" or shifting of the title/header spacer.